dev.yml

```diff
@@ -0,0 +1,5 @@
+name: web-foundation
+up:
+  - node:
+      yarn: v1.19.1
+      version: v10.17.0
```

Comment:
Can we add some [`dev commands`](https://github.com/Shopify/quilt/blob/master/dev.yml#L7-L14)